youtube_transcript_api/_api.py

```diff
@@ -61,18 +61,21 @@ def list_transcripts(cls, video_id, proxies=None, cookies=None):
         :type proxies: {'http': str, 'https': str} - http://docs.python-requests.org/en/master/user/advanced/#proxies
         :param cookies: a string of the path to a text file containing youtube authorization cookies
         :type cookies: str
+        :param verify: the path to a CA_BUNDLE file or directory with certificates of trusted CAs
```

Comment:
Probably best to link to [the official requests docs here](https://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification) as done for `proxies`.